### ISSUE TITLE
`add_epred_draws` does not show error when `re_formula =` includes non-existent terms

### ISSUE THREAD

**Issue by mz555** [mjskay/tidybayes#317]
I was using tidybayes to plot some data from a `brms` model, and realised the figure looked off, but could not tell why, until I realised the term I provided in `add_epred_draws` for `re_formula=` was misspelt.

It would be good if the package threw an error message to alert the user to this issue.

Wrong code:
`add_epred_draws(m1, re_formula = ~ (1 | item))` I used "item" with no capital. It showed no error when running the code, and only realised because my plot estimates were off.

Correct code:
`add_epred_draws(m1, re_formula = ~ (1 | Item))`

Wrong plot
![image](https://github.com/mjskay/tidybayes/assets/25532449/b22b36e1-b632-4391-aaab-3e8d19924430)
Correct plot
![image](https://github.com/mjskay/tidybayes/assets/25532449/cc745f58-9300-4234-b4f9-fdbdf5cb21c2)

**Comment by mjskay** [comment]
Yeah an error/warning here would be good. Can you check and see if you call the underlying brms function (`brms::posterior_epred`) no error/warning is generated? If that isn't generating a warning either then this is an issue for brms.
